deepwell/src/services/alias/service.rs

```diff
@@ -76,6 +76,9 @@ impl AliasService {
         // Then we check that the new slug doesn't already exist.
         // This also checks aliases, though we also verify down below that
         // it actually finds conflicts properly.
+        //
+        // If the alias is for a user, also ensures that it is over
```

Comment:
```suggestion
        // If the alias is for a user, also ensures that it is at least
```

"over" means `>`, "at least" means `>=`